freqtrade/persistence/models.py

```diff
@@ -345,6 +361,7 @@ def to_json(self) -> Dict[str, Any]:
             'max_rate': self.max_rate,
 
             'open_order_id': self.open_order_id,
+            'filled_buys': buys_json,
```

Comment:
Well, actually we might want to have all filled orders (buy and sell) here, since some people still using limit sell, and might want to have details of the sells as well

Just to be sure, we don't have `/trade <id>` currently, right? Or do I miss it?

Reply:
Apparently not for telegram (there's no reason we could not have that in the future...) - but the api has a `/trade/<id>` endpoint (although for the API, the `/trades` endpoint is the more relevant as you can get more data there).

Reply:
There is /status 123
Returns:
Trade ID: 123 (since a day ago)
Current Pair: XLM/BUSD
Amount: 9261.0 (2402.8153 BUSD)
Open Rate: 0.25945527
Current Rate: 0.26100000
Current Profit: 0.44%
Stoploss: 0.18758200 (-29.00%)
Stoploss distance: -0.07341800 (-28.13%)

Reply:
yes, but it won't work for closed trades iirc - so if trade `123` was closed, you'll not see that.

Reply:
It's still shown? That's how I get the example on the top

Reply:
/status <trade_id> works for open and closed trades as long as I have been using FT

Reply:
that's ... not incorrect apparently ...

we maybe should adjust naming a bit - showing "current_rate" in that case is quite confusing - as i'm currently not quite sure if that's the actual close rate, or today's rate (which is irrelevant for a trade closed 2 months ago).

Reply:
Yeah, we can hide `current rate` if the trade is closed. Maybe some other info as well can be hidden, like the distance of time (since xxx ago), and the stoploss and stoploss distance?

Reply:
yeah - but i'd separate that part from this PR ... as it's not really related to this change.